### ISSUE TITLE
Update Pagination Styling

### ISSUE THREAD

**Issue by yunjungyeh** [jpmorganchase/salt-ds#3758]
Figma:

Add a border around the selected page.

Code:

- [x] Add a border around the selected page.
- [x] Swap action text styles for body.
- [x] Update so selected styles override hover styles.

```[tasklist]
### Later Tasks for another issue
- [ ] Salt 1 Light/Dark
- [ ] Salt Next
- [ ] Code
```

**Dependent on issue #4332**

**Comment by mark-tate** [comment]
Latte: complete by EOS

**Comment by bhoppers2008** [comment]
Removed from sprint. Need to update #4332 first.

**Comment by yunjungyeh** [comment]
Tue 3 June - Figma components updated and merged
